apps/dashboard/src/routes/+page.svelte

```diff
@@ -89,11 +89,13 @@
 			};
 			const { error: urlError } = await data.supabase.from('urls').insert(newURL);
 			if (urlError) {
-				var msg =
-					urlError.code == '23505'
-						? `URL Prefix with the domain "${subdomain + domain}" already exists!`
-						: toastError.message;
-				toastStore.trigger({ ...toastError, message: msg });
```

Comment:
This error message needs to be moved to the call before. We can do a pass and double check any error UX before release.